docs/pages/release_notes.md

```diff
@@ -24,6 +24,27 @@ This is a {{ site.pmd.release_type }} release.
 
 ### 🚀 New and noteworthy
 
+#### New Rule UnnecessaryWarningSuppression (experimental)
+
+This new Java rule {% rule java/bestpractices/UnnecessaryWarningSuppression %} reports unused suppression
+annotations and comments. The rule class supports any PMD language,
+but language-specific behavior needs to be implemented to avoid false positives for some languages.
+Violations of this rule cannot be suppressed. It is special cased rule that is executed after all
```

Comment:
The fact it is executed after all rules is also an implementation detail and not relevant to how to use it imo